Can't install requirements for systemjs-angular-globalize

Hi,
When following the instructions on [this README](https://github.com/DevExpress/devextreme-examples/blob/17_2/systemjs-angular-globalize/README.md), I get the following error: 
```
Whops {} deepEqual { url:
    [ 'https://github.com/unicode-cldr/cldr-core/archive/30.0.3.zip',
    'https://github.com/unicode-cldr/cldr-dates-mo
npm WARN devextreme-angular@17.2.4 requires a peer of @angular/forms@>2.4.2 but none is installed. You must install peer dependencies yourself.
npm WARN devextreme-systemjs-angular-globalize@1.0.0 No repository field.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cldr-data@30.0.4 install: `node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cldr-data@30.0.4 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
Tested with Windows10, npm-5.6.0, node-v8.9.2 on branch 17_2.
Thanks in advance for any hint.

--- a/systemjs-angular-globalize/package.json
+++ b/systemjs-angular-globalize/package.json
@@ -8,7 +8,7 @@
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/common": "^5.0.0",
     "@angular/compiler": "^5.0.0",
-    "cldr-data": "^30.0.1",
+    "cldr-data": ">30.0.1",
     "devextreme-angular": "~17.2.3",
     "devextreme": "~17.2.3",
     "rxjs": "^5.0.0",

--- a/webpack-angular-globalize/package.json
+++ b/webpack-angular-globalize/package.json
@@ -8,7 +8,7 @@
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/common": "^5.0.0",
     "@angular/compiler": "^5.0.0",
-    "cldr-data": "^30.0.1",
+    "cldr-data": ">30.0.1",
     "devextreme-angular": "~17.2.3",
     "devextreme": "~17.2.3",
     "rxjs": "^5.0.0",

